fluss-server/src/test/java/com/alibaba/fluss/server/utils/TableDescriptorValidationTest.java

```diff
@@ -0,0 +1,42 @@
+package com.alibaba.fluss.server.utils;
+
+import com.alibaba.fluss.exception.InvalidTableException;
+import com.alibaba.fluss.metadata.Schema;
+import com.alibaba.fluss.types.DataTypes;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+/** Test for {@link TableDescriptorValidation} . */
+public class TableDescriptorValidationTest {
```

Comment:
Could you add an integration test `testCreateTableWithInvalidColumns` in `FlussAdminITCase`? If the IT case is added, I think we don't need this unit test anymore.